### PR TITLE
Remove patch that is no longer needed.

### PR DESCRIPTION
**Pull request by stikonas**
This fixed some issues with weak symbols but now tcc is properly patched.

### CODE REVIEW

File: steps/SHA256SUMS.pkgs [hunk 1]
```diff
@@ -20,25 +20,26 @@ e8ff13753943e43972a8cf1672819f001718ed38e5497cf7b554c21a507ba22b  automake-1.7.8
 ef08694bf8e78ffcf56f9c8a63b0f189e301c06f02ab240e15e4c7fe354c2a0d  automake-1.7_0.tar.bz2
 9922651b64ad2e29b944082f26db223008220155048f25d64df1f6d82071ff1c  automake-1.8.5_0.tar.bz2
 c5807850a377e4bb9aa4121d9781d145bdd80327626a9419630a75ce2c8d9c71  automake-1.9.6_0.tar.bz2
+5e1db0f8b00e6a3050829a296db9f6633594f6f0572825075703e074dfe1a8a7  base.tar.bz2
```
Comment (Googulator) [inline steps/SHA256SUMS.pkgs:23]: The base tarball isn't reproducible (especially between bootstrap modes), its contents are individually checksummed. It doesn't make sense to include it here.

**Review by Googulator — APPROVED** [review]
Looks good otherwise.

**Comment by Googulator** [comment]
hmm... Grub reproducibility issue?

File: steps/SHA256SUMS.pkgs [hunk 2]
```diff
@@ -59,12 +59,12 @@ c1bf792e90df13bf7a24d3f8ce3939fd96ff61b8117e4dcef75605974f3fcf25  gcc-4.0.4_1.ta
 dc67fc1b132fa3945349efe74c5b2197bd0b6babf4d29d2deddd04f09d9af680  gettext-0.21_0.tar.bz2
 309282ba840f80b5e10897b1221f3ff3329b9e05b982c788d03652f411974524  gmp-6.2.1_0.tar.bz2
 62edfa633f64202b2d29199d11d56a897654a441c7ae39a4d59ca5fe75199f61  gperf-3.1_0.tar.bz2
-276e073496931ff7d4caaabb53079412b88de953def88aec966c1f8506293e88  grep-2.4_0.tar.bz2
+c1989fc2855dbb0350c4e5b677028306614a9f4b5cb6c01fd459c5e1abf592b9  grep-2.4_0.tar.bz2
 8e612afb9a439aa8bf5db0fc2589eec890a6017690412d5c8e88a5838730b42c  grep-3.7_0.tar.bz2
-8035da0f42b40509a1f68c91164f20fec624cf8b61017aa4e135473e547dddf4  grub-2.06_0.tar.bz2
+0bf613f1f142d53baea81643383b66dcaf38438ee6ba33fb9ed1c135a0be9e47  grub-2.06_0.tar.bz2
```
Comment (Googulator) [inline steps/SHA256SUMS.pkgs:64]: This is the same unexpected hash as in the previous iteration.

Comment (stikonas) [reply steps/SHA256SUMS.pkgs:64]:
```
├── grub-2.06_0.tar
│ ├── file list
│ │ @@ -638,15 +638,15 @@
│ │  -rw-r--r--   0        0        0     3197 1970-01-01 00:00:00.000000 ./usr/share/man/man1/grub-mkrescue.1
│ │  -rw-r--r--   0        0        0     2878 1970-01-01 00:00:00.000000 ./usr/share/man/man1/grub-mkstandalone.1
│ │  -rw-r--r--   0        0        0     1388 1970-01-01 00:00:00.000000 ./usr/share/man/man1/grub-render-label.1
│ │  -rw-r--r--   0        0        0      907 1970-01-01 00:00:00.000000 ./usr/share/man/man1/grub-script-check.1
│ │  -rw-r--r--   0        0        0     1807 1970-01-01 00:00:00.000000 ./usr/share/man/man1/grub-syslinux2cfg.1
│ │  drwxr-xr-x   0        0        0        0 1970-01-01 00:00:00.000000 ./usr/share/man/man8/
│ │  -rw-r--r--   0        0        0     2005 1970-01-01 00:00:00.000000 ./usr/share/man/man8/grub-bios-setup.8
│ │ --rw-r--r--   0        0        0     4264 1970-01-01 00:00:00.000000 ./usr/share/man/man8/grub-install.8
│ │ +-rw-r--r--   0        0        0     4267 1970-01-01 00:00:00.000000 ./usr/share/man/man8/grub-install.8
│ │  -rw-r--r--   0        0        0      990 1970-01-01 00:00:00.000000 ./usr/share/man/man8/grub-macbless.8
│ │  -rw-r--r--   0        0        0      872 1970-01-01 00:00:00.000000 ./usr/share/man/man8/grub-mkconfig.8
│ │  -rw-r--r--   0        0        0      566 1970-01-01 00:00:00.000000 ./usr/share/man/man8/grub-ofpathname.8
│ │  -rw-r--r--   0        0        0     1628 1970-01-01 00:00:00.000000 ./usr/share/man/man8/grub-probe.8
│ │  -rw-r--r--   0        0        0     1661 1970-01-01 00:00:00.000000 ./usr/share/man/man8/grub-reboot.8
│ │  -rw-r--r--   0        0        0     1124 1970-01-01 00:00:00.000000 ./usr/share/man/man8/grub-set-default.8
│ │  -rw-r--r--   0        0        0     2026 1970-01-01 00:00:00.000000 ./usr/share/man/man8/grub-sparc64-setup.8
│ ├── ./usr/share/man/man8/grub-install.8
│ │ @@ -112,15 +112,15 @@
│ │  is only available on EFI.
│ │  .TP
│ │  \fB\-s\fR, \fB\-\-skip\-fs\-probe\fR
│ │  do not probe for filesystems in DEVICE
│ │  .TP
│ │  \fB\-\-target\fR=\fITARGET\fR
│ │  install GRUB for TARGET platform
│ │ -[default=i386\-pc]; available targets:
│ │ +[default=x86_64\-efi]; available targets:
│ │  arm\-coreboot, arm\-efi, arm\-uboot, arm64\-efi,
│ │  i386\-coreboot, i386\-efi, i386\-ieee1275,
│ │  i386\-multiboot, i386\-pc, i386\-qemu, i386\-xen,
│ │  i386\-xen_pvh, ia64\-efi, mips\-arc, mips\-qemu_mips,
│ │  mipsel\-arc, mipsel\-loongson, mipsel\-qemu_mips,
│ │  powerpc\-ieee1275, riscv32\-efi, riscv64\-efi,
│ │  sparc64\-ieee1275, x86_64\-efi, x86_64\-xen
```

Comment (stikonas) [reply steps/SHA256SUMS.pkgs:64]: https://github.com/fosslinux/live-bootstrap/issues/463

**Pull request merged**